phantom.coffee

```diff
@@ -61,7 +61,7 @@ module.exports =
     if typeof options.parameters is 'object'
       for key, value of options.parameters
         args.push '--'+key+'='+value
-    options.path ?= ''
+    options.path ?= './node_modules/phantomjs/bin/'
```

Comment:
@amir20,
I've assumed that if there is no path set at this point (vanilla installation), we switch to the prebuilt binary (that is './node_modules/phantomjs/bin/phantomjs') instead of a system binary (just 'phantomjs').

But it is my first contact with this codebase, so I'm happy to hear your thoughts - how would you like to have this handled?